apps/wallet/src/ui/app/pages/home/apps/index.tsx

```diff
@@ -66,6 +67,10 @@ function AppsPage() {
 		await setToSessionStorage<string>(APPS_PAGE_NAVIGATION, tag.link);
 	};
 
+	if (useUnlockedGuard()) {
```

Comment:
won't this hook redirect to the `/tokens` screen? do we need to `return null` everywhere if we're going to redirect when the account is locked?

Reply:
It has a loading state, mainly until accounts are loaded. If that takes long returning null will avoid showing any contents of the actual page